src/bsl/features/signatureHelpProvider.ts

```
@@ -1,58 +1,132 @@
 import { editor, languages, Position, CancellationToken } from 'monaco-editor-core'
 import { scopeProvider } from '../scopeProvider'
-import { Symbol, SymbolType, MethodSymbol, MethodSignature, isPlatformMethod } from '../../scope'
+import { Symbol, SymbolType, MethodSymbol, MethodSignature, isPlatformMethod, EditorScope, GlobalScope } from '@/scope'
```

Biome (1.9.4)

[error] Do not shadow the global "Symbol" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)